src/System.Private.ServiceModel/pkg/System.Private.ServiceModel.pkgproj

```diff
@@ -3,6 +3,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <PreventImplementationReference>true</PreventImplementationReference>
+    <!-- workaround for https://github.com/dotnet/buildtools/issues/1529 -->
+    <HarvestVersion>4.3.0</HarvestVersion>
```

Comment:
This was fixed in latest buildtools.  I would have expected you to be on that (corefx is)